tests/NPSP.robot

```diff
@@ -61,7 +61,21 @@ API Create Secondary Affiliation
     ...               npe5__Contact__c=${contact_id}
     ...               npe5__Primary__c=false 
     ...               &{fields}
-  
+ 
+# API Create Engagement Plan
+    # [Arguments]      ${plan_name}     &{fields}    
+    # ${opp_id} =  Salesforce Insert    npsp__Engagement_Plan_Template__c
+    # ...               Name=${plan_name}
+    # ...               npsp__Description__c=This plan is created via Automation 
+    # ...               &{fields}
+   
+API Create GAU
+    ${name} =   Generate Random String
+    ${gau_id} =  Salesforce Insert  npsp__General_Accounting_Unit__c
+    ...               Name=${name}
+    &{gau} =     Salesforce Get  npsp__General_Accounting_Unit__c  ${gau_id}
+    [return]         &{gau}  
```

Comment:
We need to use a variable for the namespace prefix (`npsp__`) because the object does not have that prefix when NPSP is installed as an unmanaged package. But I need to do some work to figure out how we should set that variable correctly based on how NPSP was installed, so it's okay if that waits until later.